plugins/kafka/src/components/ConsumerGroupOffsets/useConsumerGroupsForEntity.ts

```diff
@@ -32,12 +63,18 @@ export const useConsumerGroupsForEntity = () => {
           `Failed to parse kafka consumer group annotation: got "${annotation}"`,
         );
       }
+
       return {
         clusterId: clusterId.trim(),
         consumerGroup: consumerGroup.trim(),
+        dashboardUrl: getDashboardUrl(
```

Comment:
Are dashboards able to display individual consumer groups, and should we be able to link straight to them? Feeling we're missing out on that possibility with this way of structuring the data.

Reply:
I think it depends on the dashboard product but for AKHQ or Kowl there is a deep link feature.

I could imagine that we use <cluster>/<consumer-group-name>/<dashboard-url> for the annotation, this will make the consumer group name, in the table header, a link and add a button in the table header for the dashboard url, which is configured in the app-config.yaml.

What do you think?

Reply:
Hmm, feels like it gets a lil bit hacky and rigid. Might be worth introducing a level of indirection through a Utility API?

Thinking an API with a single `getDashboardUrl(entity)` or similar. Then a default implementation which pretty much mirrors the existing logic, and perhaps a few helpers to create more advanced URLs

Reply:
Picked up work again 😉 
I think it's a good idea to have a API like `kafkaDashboardApiRef` to hide the complexity and give flexibility for future implementations.

Never the less I would recommend to have a 1:1 relation between cluster and a dashboard url, as I think you will most probably not deploy a dashboard tool per consumer group. Especially if the consumer groups are all owned/used by the same component.

What do you think @Rugvip ?